addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23405,3 +23405,8 @@ msgstr ""
 msgctxt "#39189"
 msgid "Available only with manual subtitle position"
 msgstr ""
+
+#: xbmc/playlists/SmartPlaylist.cpp
+msgctxt "#39190"
+msgid "Unique ID value"
```

Comment:
I'm not sure if `value` is really necessary here.

Reply:
If you feel strongly about it, I can remove, but I do think it should be there. A unique ID has two parts, ID (tmdb, imdb, etc) and a value (the actual key value). In this case the filter is on the value.